Mathlib/Analysis/Convex/ProjIcc.lean

```diff
@@ -0,0 +1,99 @@
+/-
```

Comment:
This file is new, right? If so, can you drop it from this PR so that   it can be reviewed as a port, not a forward-port?

Reply:
This file *is* a forward-port, though?

Reply:
It's only a forward-port if it's a modification of a file that was already ported from the out-of-sync region of https://leanprover-community.github.io/mathlib-port-status; otherwise it's just a sparkling port.

It's best to do this in a separate PR using `start_port.sh` so that the review against mathport is possible. Obviously that will have to wait for the rest of this files in this PR to be merged first.

Reply:
Did you recently change the presentation of `out-of-sync`? I swear I copied that code from there 🤔

Reply:
I haven't touched it...